.github/workflows/deb_release.yml

```diff
@@ -0,0 +1,73 @@
+name: Debian packages release
+
+on:
+  workflow_dispatch:
+    inputs:
+      config:
+        description: JSON of options
+        type: string
+        default: '{}'
+        required: true
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save the config options
+        run: |
+          echo '${{ inputs.config }}' | jq > config.json
+      - name: Archive the release configuration
+        uses: actions/upload-artifact@v3
+        with:
+          name: config.json
+          path: config.json
+      - name: Install dependencies
+        run: |
+          sudo apt update -qq
+          sudo apt install -qq -y bumpversion devscripts python3-launchpadlib
+          git clone https://git.launchpad.net/~checkbox-dev/checkbox/+git/support
```

Comment:
This is a part of Checkbox, so it feels like having this outside monorepo defeats the purpose of monorepo.

Reply:
deb release scripts are now part of the monorepo (tools/release/)